etc/vcl_snippets/recv.vcl

```diff
@@ -115,7 +115,7 @@
         # and allow custom parameters to be set. List of parameters is configurable in admin
         set req.http.Magento-Original-URL = req.url;
         # Change the list of ignored parameters by configuring them in the Advanced section
-        set req.url = querystring.regfilter(req.url, "^(####QUERY_PARAMETERS####)");
+        set req.url = querystring.regfilter(req.url, "^(####QUERY_PARAMETERS####)(=|&|$)");
```

Comment:
I think this can be simplified to:

```
set req.url = querystring.regfilter(req.url, "^(####QUERY_PARAMETERS####)$");
```

Reply:
I believe that's right.

Reply:
Thanks for the feedback; I've applied this change.